project/firstproject/src/main/java/com/example/firstproject/repository/AnswerRepository.java

```diff
@@ -3,7 +3,13 @@
 import com.example.firstproject.model.Answer;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
 
 public interface AnswerRepository extends JpaRepository<Answer, Long>
 {
+    @Query(value = "SELECT * FROM Answer a WHERE a.question_id = :id ", nativeQuery = true)
+    List<Answer> listAnswers(@Param("id") Long questionId);
```

Comment:
Minor: Change the method name with more descriptive name like "listAnswersByQuestion" or something similar.